test/test_jld2_output_writer.jl

```diff
@@ -58,7 +60,7 @@ function test_jld2_file_splitting(arch)
                           including = [:grid],
                           array_type = Array{Float64},
                           with_halos = true,
-                          file_splitting = FileSizeLimit(200KiB),
+                          file_splitting = filesizelimit,
```

Comment:
```suggestion
                          file_splitting = file_size_limit,
```

I like how it was previously though, faster to read...